spec/openapi.yaml

```diff
@@ -92,38 +106,47 @@ paths:
               $ref: '#/components/schemas/AddKeywordsRequest'
       responses:
         '200':
-          description: Successfully updated
+          description: Keyword(s) successfully added.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/AddKeywordsResponse'
+        '400':
+          description: Invalid request. Tag name and additional keywords are required.
+        '401':
+          description: A valid API key is required to access this resource.
+        '404':
+          description: Tag not found.
+        '500':
+          description: An unexpected error occured.
 
-  /tag/{tagName}/keyword/{keyword}:
```

Comment:
`tagName` to be supplied as a field in JSON payload. To make things consistent, have also moved keyword to JSON payload.